test/donotoptimize_assembly_test.cc

```diff
@@ -38,88 +38,100 @@ const ExtraLarge ConstExtraLargeObj{};
 // CHECK-LABEL: test_with_rvalue:
 extern "C" void test_with_rvalue() {
   benchmark::DoNotOptimize(Add42(0));
-  // CHECK: movl $42, %eax
-  // CHECK: ret
+  // CHECK-CLANG: mov 2a,REG
```

Comment:
Forgot about this one. It should be `CHECK: mov 2a,REG` not `CHECK-CLANG: mov 2a,REG`

Reply:
Fixed in new commit